internal/config/config.go

```diff
@@ -78,86 +60,54 @@ func (c Config) Update(kvs []string) (Config, []string, error) {
 		}
 	}
 
-	for i, a := range kvs {
+	var currField string
+	for i, v := range kvs {
 		if i%2 == 0 {
-			c.RawConfig[a] = struct{}{}
-			updatedFields = append(updatedFields, a)
+			currField = v
```

Comment:
Since the CLI gets the input as a slice of strings, the even-numbered values are keys and the odd-numbered values are values.